.gitignore

```diff
@@ -3,8 +3,7 @@
 .idea/
 out/
 target/
-java/
```

Comment:
why did you remove this? java typically contains javadocs which I dont think we want to check in.